In-house patch to remove amqplib from Cinder's requirements files as
an alternate implementation is used on Solaris.

--- cinder-2013.1.4/cinder.egg-info/requires.txt.orig 2013-11-22 07:46:37.051978903 -0700
+++ cinder-2013.1.4/cinder.egg-info/requires.txt        2013-11-22 07:46:45.304481983 -0700
@@ -1,5 +1,4 @@
 SQLAlchemy>=0.7.3,<=0.7.9
-amqplib>=0.6.1
 anyjson>=0.2.4
 eventlet>=0.9.17
 kombu>=1.0.4

--- cinder-2013.1.4/tools/pip-requires.orig       2013-11-22 07:48:29.381781247 -0700
+++ cinder-2013.1.4/tools/pip-requires        2013-11-22 07:48:34.137850426 -0700
@@ -1,5 +1,4 @@
 SQLAlchemy>=0.7.3,<=0.7.9
-amqplib>=0.6.1
 anyjson>=0.2.4
 argparse
 eventlet>=0.9.17
